bat-utils/lib/extras-utils.js

```diff
@@ -2,13 +2,38 @@ const getPublisherProps = require('bat-publisher').getPublisherProps
 // this can be abstracted elsewhere as soon as we finish #274
 const BigNumber = require('bignumber.js')
 const dotenv = require('dotenv')
+const Bottleneck = require('bottleneck/es5')
 dotenv.config()
 BigNumber.config({
   EXPONENTIAL_AT: 28,
   DECIMAL_PLACES: 18
 })
 
+const ledgerLimiter = new Bottleneck({
+  reservoir: 50, // initial value
+  reservoirRefreshAmount: 25,
+  reservoirRefreshInterval: 10 * 1000, // must be divisible by 250
+  // also use maxConcurrent and/or minTime for safety
+  maxConcurrent: 32,
+  minTime: 0.1
+})
+
+const upholdLimiter = new Bottleneck({
```

Comment:
these limits are per instance - correct?

Reply:
yes. they would end up being per instance. i believe i can set it up to be shared across a redis store as well if that is preferrable